docs/nodes/arabica-devnet.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_label : Arabica Devnet
+---
+
+# Arabica Devnet
+<!-- markdownlint-disable MD013 -->
+
+![arabica-devnet](/img/mamaki.png)
+
+Arabica Devnet is a new testnet from Celestia Labs focused exclusively
+on deploying the latest updates from the Celestia Labs team to a
+network that will always have the latest changes.
+
+Arabica does not focus on validator or consensus-level testing, rather,
+that is what Mamaki Testnet is used for. If you are a validator, we
+recommend just testing your validator operations on Mamaki [here](./mamaki-testnet.md).
+
+Arabica instead focuses exclusively on developers, hence it being a devnet.
+With Arabica having the latest updates from all Celestia's products deployed
+on it, it can be subject to many changes. Therefore, as a fair warning,
+Arabica can break unexpectedly but given it will be continously updated,
+it is a useful way to keep testing the latest changes in the software.
+
+Developers can still deploy on Mamaki Testnet their sovereign rollups if they
+chose to do so, it just will always lag behind Arabica Devnet until Mamaki
+undergoes Hardfork Upgrades in coordination with Validators.
```

Comment:
The first paragraph should instead explicitly state what the devnet will be for (which you did in the third pararaph). So, the first paragraph could be something like "Arabica Devnet is a new testnet from Celestia Labs that is focused exclusively on providing developers with enhanced performance and the latest upgrades for testing their rollups and applications."

Second paragraph is good. Then the third paragraph just needs to remove the first sentence since it will get mentioned earlier.